### PR TITLE
Pyocf minor api fixes

### PR DESCRIPTION
**Pull request by KamilLepek**
Signed-off-by: Kamil Lepek <kamil.lepek94@gmail.com>

### CODE REVIEW

**Comment by KamilLepek** [comment]
retest this please

**Comment by KamilLepek** [comment]
retest this please

File: tests/functional/pyocf/utils.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def TiB(self):
 
     @property
     def sectors(self):
-        return self.bytes // _SECTOR_SIZE
```
Comment (arutk) [inline tests/functional/pyocf/utils.py]: double slash was intentional -  this is integer division with rounding down

**Comment by KamilLepek** [comment]
retest this please

**Pull request merged**